pkg/apis/core/v1alpha1/conversions.go

```diff
@@ -1699,6 +1710,19 @@ func Convert_v1alpha1_ShootStatus_To_garden_ShootStatus(in *ShootStatus, out *ga
 	if err := autoConvert_v1alpha1_ShootStatus_To_garden_ShootStatus(in, out, s); err != nil {
 		return err
 	}
+	if in.LastError != nil {
+		outLastErrors := []garden.LastError{
+			garden.LastError{
```

Comment:
you don't need to write `garden.LastError here`
```suggestion
			{
```